director/ssh_opts_test.go

```diff
@@ -13,21 +15,43 @@ import (
 )
 
 var _ = Describe("NewSSHOpts", func() {
+	const UUID = "2a4e8104-dc50-4ad7-939a-2efd53b029ae"
+	const ExpUsername = "bosh_2a4e8104dc504ad"
+
 	var (
 		uuidGen *fakeuuid.FakeGenerator
 	)
 
 	BeforeEach(func() {
 		uuidGen = &fakeuuid.FakeGenerator{
-			GeneratedUUID: "2a4e8104-dc50-4ad7-939a-2efd53b029ae",
+			GeneratedUUID: UUID,
+		}
+	})
+
+	It("generates a username that is compatible with Windows", func() {
+		const MaxLength = 20
+		const InvalidChars = "\"/\\[]:|<>+=;?*"
```

Comment:
Nice additional test coverage. If there's an official list of invalid chars somewhere, it'd be helpful to include a comment/link for more documentation.